apps/system-e2e/src/tests/judicial-system/smoke/shared-steps/send-appeal.ts

```diff
@@ -30,6 +30,27 @@ export async function prosecutorAppealsCaseTest(page: Page, caseId: string) {
     verifyRequestCompletion(page, '/api/graphql', 'CreateFile'),
   ])
 
+  const appealCaseFileChooserPromise = page.waitForEvent('filechooser')
+  await page
+    .locator('section')
+    .filter({
+      hasText:
+        'GögnEf ný gögn eiga að fylgja kærunni er hægt að hlaða þeim upp hér að neðan. At',
+    })
```

Comment:
Could use better selector here.

Reply:
You're right, I did a bit of cleanup and refactoring to make this and also file uploading in general cleaner.